version/constants.go

```diff
@@ -74,41 +74,111 @@ var (
 	// by avalanchego, but is useful for downstream libraries.
 	RPCChainVMProtocolCompatibility map[uint][]*Semantic
 
+	DefaultUpgradeTime    = time.Date(2020, time.December, 5, 5, 0, 0, 0, time.UTC)
+	unreachableFutureTime = time.Date(1000, time.January, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
1000 is already in the past. should it be 1000 or 10000 ?

Reply:
ah, sry, intended to set it to 10000